data/org.freedesktop.portal.Usb.xml

```diff
@@ -0,0 +1,265 @@
+<?xml version="1.0"?>
+<!--
+ Copyright (C) 2020 Endless OS Foundation LLC
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2 of the License, or (at your option) any later version.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public
+ License along with this library. If not, see <http://www.gnu.org/licenses/>.
+
+ Author: Ryan Gonzalez <rymg19+github@gmail.com>
+-->
+
+<node name="/" xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
+  <!--
+      org.freedesktop.portal.Usb:
+      @short_description: Portal for USB device access
+
+      This interface lets sandboxed applications monitor and request
+      access to connected USB devices.
+
+      This documentation describes version 1 of this interface.
+    -->
+  <interface name="org.freedesktop.portal.Usb">
+    <!--
+        EnumerateDevices:
+        @options: Vardict with optional further information
+        @devices: Dictionary of device IDs to a vardict of device properties.
+                  See org.freedesktop.portal.Usb.GetDeviceProperties() for a list
+                  of all the properties that may be present in the vardicts.
+
+        Enumerates all connected USB devices that this application has permission
+        to see.
+
+        Supported keys in the @options vardict include:
+        <variablelist>
+          <varlistentry>
+            <term>filter a{sv}</term>
+            <listitem><para>
+              A vardict of device properties that is used to filter the enumerated devices.
+              Each property value in the filter must exactly match the corresponding property
+              values in the returned devices.
+            </para></listitem>
+          </varlistentry>
+        </variablelist>
+     -->
+    <method name="EnumerateDevices">
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="a{sa{sv}}" name="devices" direction="out"/>
+    </method>
+
+    <!--
+        GetDeviceProperties:
+        @id: Device ID to query the properties of.
+        @device: Device properties attached to the ID.
+
+        Finds all the known properties associated with the given device.
+
+        Supported keys in the @device vardict include:
+        <variablelist>
+          <varlistentry>
+            <term>parent s</term>
+            <listitem><para>
+              Device ID of the parent device.
+            </para></listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>devnode s</term>
```

Comment:
Perhaps `device_node`? (Probably not a good idea though, since `devnode` matches udev terminology)